website/content/en/highlights/2023-09-26-0-33-0-upgrade-guide.md

```diff
@@ -34,9 +40,23 @@ with the other Datadog sinks, which use the `endpoint` as a base URL that the AP
 With this release, the `datadog_logs` sink's behavior is now consistent with the other
 Datadog sinks for the `endpoint` setting.
 
+#### Disable OpenSSL legacy provider by default {#openssl-legacy-provider}
+
+Vector upgraded the version of OpenSSL that it statically compiles in to v3.1.x in the 0.32.0 release.
+Following our deprecation policy, v0.33.0 now disables the legacy OpenSSL provider by default. It can be
+enabled through the [OpenSSL configuration options](https://www.openssl.org/docs/manmaster/man5/config.html)
+exposed through environment variables, particularly `OPENSSL_CONF`.
```

Comment:
The only concrete and relevant env variable is `OPENSSL_CONF` to specify an OpenSSL config file to use.
I think mentioning "variables" and "particularly" is vague, misleading and confusing.

While outside of the scope of this PR, I still think Vector should document a complete example like I gave in the other closed PR. That should make it very clear how to use this functionality.

Reply:
There is one more thing to consider maybe.

Vector's vendored OpenSSL library actually looks by default for `/usr/local/ssl/openssl.cnf`, which likely does not exist in most user's systems by default. The `OPENSSL_CONF` variable actually allows to override this to wherever you need.

So, technically, if a user has an OpenSSL config in that path, Vector will pick it up automatically too. Which can be suprising behaviour for them (maybe). Therefore, I'm not sure if this is to be mentioned somewhere too.

Perhaps a dedicated "Vector and SSL/TLS" documentation section is necessary to explain all these nuances?
Examples of using `OPENSSL_CONF` could go there for instance.

Reply:
I agree that it would be useful to go into detail on this topic in the documentation. Given your expertise in the area, is this something you'd be willing to contribute? If not, I can open an issue to track this work.

Reply:
As I mentioned in the other PR where I investigated all of this, I'm more than happy to contribute documentation on the subject. As a starting point, can the Vector team suggest where in the documentation it would be suitable to open a new section/page for this topic? 

At the moment it is hard for me to see where it would best fit in <https://vector.dev/docs/> :(

Reply:
Thanks! I think this topic is worthy of its own page - `https://vector.dev/docs/reference/configuration/tls` seems like a good location. You can add this by creating a new markdown file similar to https://github.com/vectordotdev/vector/blob/master/website/content/en/docs/reference/configuration/unit-tests.md

Reply:
Thanks for the pointer! Will work on it in the coming days.
I have one more ask, how can I quickly generate locally a render of the docs website?

Reply:
You should be able to run `make serve` in the `website` directory to spin up a local instance.